packages/opentelemetry-metrics/src/export/Controller.ts

```diff
@@ -38,8 +38,13 @@ export class PushController extends Controller {
     unrefTimer(this._timer);
   }
 
-  private _collect() {
-    this._meter.collect();
+  shutdown() {
+    this._collect();
```

Comment:
Why does shutdown call an async collect but not await it?

Reply:
Could you give an example of when this could have negative side effects? I was under the impression that async functions could continue to run even if the parent function had finished.

Reply:
It's not that it will have a negative side-effect, but that some users may want to manually call shutdown and wait for it to finish.

Reply:
I can make shutdown async as well and add awaits so users can wait for it to finish if they call it manually.